docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -28,6 +28,33 @@ _[Subscriptions](/azure/cloud-adoption-framework/ready/landing-zone/design-area/
 
 An Azure _resource group_ is a container that holds related resources for an Azure solution. To learn how to manage resource groups within your subscription, see [How to manage Azure resource groups with the Azure CLI](manage-azure-groups-azure-cli.md)
 
+## Access tokens 
```

Comment:
```suggestion
## Refresh tokens
```

Reply:
Opon more consideration, please move this section to authenticate-azure-cli.md.  Access tokens are about sign in, not about managing subscriptions.